ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -186,10 +187,30 @@ public void shouldCleanUpSchemaRegistry() throws Exception {
 
     assertThat(schemaRegistryClient.getAllSubjects(), hasItem("BAR-value"));
     ksqlEngine.terminateQuery(new QueryId("CTAS_BAR_0"), true);
-    ksqlEngine.buildMultipleQueries("DROP TABLE bar;", Collections.emptyMap());
+    ksqlEngine.buildMultipleQueries("DROP TABLE bar DELETE TOPIC;", Collections.emptyMap());
+    assertThat(topicClient.isTopicExists("BAR"), equalTo(false));
     assertThat(schemaRegistryClient.getAllSubjects().contains("BAR-value"), equalTo(false));
   }
 
+  @Test
+  public void shouldNotDeleteSchemaFromSchemaRegistry() throws Exception {
```

Comment:
This should be renamed to `shouldNotDeleteSchemaNorTopic`. Also add separate tests for `drop stream`.